composer.json

```diff
@@ -23,9 +23,7 @@
   "require"    : {
     "php": ">=5.4.0",
     "consolidation/robo": "~1",
-    "joomla/github": "~1.3",
-    "greencape/coding-standards": "~1",
```

Comment:
Interesting, for me it just has conflicts with latest Jorobo and Selenium in weblinks.. (Working on updating the tests there and do a switch to chrome)